apps/email/js/cards/message_list.js

```diff
@@ -533,6 +534,7 @@ return [
       // position in case the user has scrolled before this code
       // runs.
       if (searchBar.classList.contains('collapsed')) {
+        this.searchButton.disabled = true;
```

Comment:
We do not need this call here, but in _cacheDom, we will want to be sure the disabled attribute is set on the node that will be saved to the cache.